### DISCUSSION TITLE
Weekly Recommenders meeting

### DISCUSSION THREAD

**Discussion by miguelgfierro** [recommenders-team/recommenders#2013] [Announcements]
Our weekly meetings are open to anyone who wants to participate:

Every Monday 5-5.30pm CET.

Microsoft Teams meeting
Join on your computer, mobile app or room device
[Click here to join the meeting](https://teams.microsoft.com/l/meetup-join/19%3acf53f267b15340de8867aa869ced4ed1%40thread.skype/1673456128585?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%22c798428d-eb18-456e-aed9-c2ecae3a8510%22%7d)

Meeting ID: 210 714 170 875 

Passcode: EoZTGC 

[Download Teams](https://www.microsoft.com/en-us/microsoft-teams/download-app) | [Join on the web](https://www.microsoft.com/microsoft-teams/join-a-meeting)

**Comment by ChenglongMa** [comment]
Hi,

Thanks for the great news!

Could I know the agenda or the topic for each week of the meeting?

Thanks!

**Reply by miguelgfierro** [reply]
It will change every week. Typically it is related to the most urgent developments we need to do in the repo.

**Reply by ChenglongMa** [reply]
Got it. Thanks!
